### ISSUE TITLE
2.0.0 release checklist

### ISSUE THREAD

**Issue by patr1ckm** [civisanalytics/civis-r#179]
- [x] update default client
- [x] integration tests
- [x] docs
- [x] cran

**Comment by keithing** [comment]
I think we also want to run integration tests

**Comment by HenrikBengtsson** [comment]
Hi, where are you guys when it comes to a CRAN release?  Some guestimate for an ETA? I'm interested in rolling out a cleaned-up version of future, but that requires the #168 fix to be on CRAN.

**Comment by keithing** [comment]
@patr1ckm how close do you think we are to releasing 2.0?  Should we consider releasing 1.7 in the meantime to exposes #168?

**Comment by patr1ckm** [comment]
Close! I've been testing `dev-2.0` on my machine for a few weeks, so we should be ready to go this week. I'll finish out the integration tests and tag you.

**Comment by HenrikBengtsson** [comment]
FYI, I've just checked with the [develop branch of future](https://github.com/HenrikBengtsson/future/tree/develop) - soon to be released, and `dev-2.0` passes `R CMD check --as-cran` on my notebook(*).  The main update there is that all 'condition' classes will be captured and relayed; previously only 'message' and 'warning' (+ 'error') were relayed.

(*) with the following constraint:
```r
Generating API
The environmental variable CIVIS_API_KEY is not set. Add this to your .Renviron or call Sys.setenv(CIVIS_API_KEY = '<api_key>')
Generating API failed, reverting to default.
```

**Comment by patr1ckm** [comment]
We should be good to go!
